### DISCUSSION TITLE
Hysteria 2  使用几天会出现failed (code=killed, signal=KILL)

### DISCUSSION THREAD

**Discussion by lin878583** [apernet/hysteria#1085] [Q&A]
Hysteria 2  使用几天会出现failed， (code=killed, signal=KILL)
要使用systemctl start hysteria-server.service  会恢复正常，请问这是为什么啊
![8888](https://github.com/apernet/hysteria/assets/168949164/c70586a7-1d9c-468c-803c-3275ab3901a0)
![9999](https://github.com/apernet/hysteria/assets/168949164/56e63ea6-2668-43e1-a705-a87d0097e9d1)

**Comment by haruue** [comment]
通常是内存占用过多被系统 LMK 了。
v2.4.3 在部分环境下有内存泄漏问题， 可以试试回退到 [v2.4.1](https://github.com/apernet/hysteria/releases/tag/app%2Fv2.4.1) 。

**Reply by lin878583** [reply]
谢谢啊，我怎么能确定我目前的是v2.4.3，新手别多见谅啊

**Reply by lin878583** [reply]
怎么回退啊

**Reply by lin878583** [reply]
大佬我用的nekobox 客户端

**Reply by haruue** [reply]
https://github.com/apernet/hysteria/issues/1086#issuecomment-2116623405
